README.md

```diff
@@ -103,18 +103,22 @@ To install Cilium while automatically detected:
 
     cilium status
         /¯¯\
-     /¯¯\__/¯¯\    Cilium:      OK
-     \__/¯¯\__/    Operator:    OK
-     /¯¯\__/¯¯\    Hubble:      OK
-     \__/¯¯\__/
-        \__/
+     /¯¯\__/¯¯\    Cilium:          OK
+     \__/¯¯\__/    Operator:        OK
+     /¯¯\__/¯¯\    Envoy:           OK
```

Comment:
If Envoy DaemonSet isn't enabled in cilium, the status prints `Envoy: disabled`. 

This only refers to the "Envoy as DaemonSet" deployment - in this case Envoy still gets executed as process within the Cilium Agent container if necessary. At least as long as this "embedded" mode is supported.

Therefore, this output might be irritating for the user.

Reply:
Right, especially if the have Envoy installed in other contexts.

Can you just omit this if disabled?

Reply:
@squeed not without having to change the logic of `statusSummary` or conditionally wrap the call to it.

do you think it would already help to change the label to something like `Ext. Envoy`?

Reply:
@mhofstetter IMO it would be better if this would somehow indicate that envoy is started on demand in the embedded mode, as "Envoy: disabled" is very misleading - there is no functional difference even if "Envoy: disabled" is shown!

Maybe changing the label to `Envoy daemonset` would be enough? I think "Ext. Envoy" brings in too much detail about our Envoy deployment model(s), especially if we deprecate the embedded mode at some point.

Reply:
@jrajahalme i changed the label to `Envoy DaemonSet`

agree that it would be nice to see whether the embedded envoy has been started on demand or not. but this is kind of another topic - which should also be reflected in the `cilium status` within the Cilium Agent first.

Reply:
I chatted with @mhofstetter, and I suggested writing "Disabled (using embedded mode)" to prevent any possible confusion.